documentation/staging/content/userguide/kubernetes/k8s-setup.md

```diff
@@ -40,8 +40,8 @@ Follow the basic steps from the  [Terraform Kubernetes installer for Oracle Clou
 #### Prerequisites
 
 1. Download and install [Terraform](https://www.terraform.io/) (v0.10.3 or later).
-2. Download and install the [OCI Terraform Provider](https://github.com/terraform-providers/terraform-provider-oci/releases) (v2.0.0 or later).
-3. Create an Terraform configuration file at  `~/.terraformrc` that specifies the path to the OCI provider:
+2. Download and install the [Oracle Cloud Infrastructure Terraform Provider](https://github.com/terraform-providers/terraform-provider-oci/releases) (v2.0.0 or later).
```

Comment:
Drop "/releases" from this URL as they have changed how they distribute this technology (just use https://github.com/terraform-providers/terraform-provider-oci). Also, it looks like it describes itself as "Terraform Provider for Oracle Cloud Infrastructure".